proto/src/lib.rs

```diff
@@ -29,6 +29,16 @@ pub mod cosmos {
     pub mod auth {
         pub mod v1beta1 {
             include!("prost/cosmos.auth.v1beta1.rs");
+            /// EthAccount defines an Ethermint account.
+            /// TODO: remove when https://github.com/cosmos/cosmos-sdk/pull/9981
```

Comment:
I think that PR only address the key derivation, not adding the account type, so even with that merged the `EthAccount` is still needed here?

Reply:
oh yeah -- it only has proto for PubKey: https://github.com/cosmos/cosmos-sdk/pull/9981/files#diff-8536674359a7a42dac3c0752b0671a6ead8a834682c0aa95463f978d6185da7e